pkg/api/schema.graphql

```diff
@@ -0,0 +1,83 @@
+schema {
+    query: Query
+}
+
+# Leave date as a string (specify YYYY-MM-DD) and convert later?
+# How do we do latest/earliest?
+#scalar Date
+
+# Query contains the entry points into the schema
+type Query {
+    #bug(id: Int!, datestamp: String = "latest"): Bug
```

Comment:
We currently don't have a use case for fetching a single bug - but it would be very easy to implement and could be useful for other users of the API.